src/components/ReportActionItem/IOUPreview.js

```diff
@@ -136,32 +130,32 @@ const IOUPreview = (props) => {
     if (props.iouReport.total === 0) {
         return null;
     }
-
     const sessionEmail = lodashGet(props.session, 'email', null);
     const managerEmail = props.iouReport.managerEmail || '';
     const ownerEmail = props.iouReport.ownerEmail || '';
+    const participantEmails = props.isBillSplit ? props.action.originalMessage.participants : [managerEmail, ownerEmail];
```

Comment:
Just a question since I am not sure how the IOU action is built. When it's a 1:1 action, we don't store the managerEmail and ownerEmail in participants? Just wondering why we can't use participants for both cases.

Reply:
I agree with the conclusion from Jules

Reply:
Thank you for the explanation!